test/PHPMailer/DKIMTest.php

```diff
@@ -79,15 +93,15 @@ public function testDKIMHeaderCanonicalization()
      */
     public function testDKIMOptionalHeaderFieldsCopy()
     {
-        $privatekeyfile = 'dkim_private.pem';
+        $privatekeyfile = self::PRIVATE_KEY_FILE;
```

Comment:
Any particular reason not to use the constant directly instead of going via a var?

Reply:
Only for less code churn, but happy to change it.

Reply:
Done in the applicable commit..